### PR TITLE
 Change UI for methods bank

### PR DESCRIPTION
**Pull request by shubhi1092**
This changes the look and feels of the spell palette view (methods bank) according to ozaria. It does not affect anything for the existing play level view and hence is safe to ship.
Also a lot of sass files have just been moved from legacy to ozaria folder and do not need a review (basically only second commit needs a review).

Screenshots of the new methods bank (this is the initial design implementation, I am sure there will be some polishing to it later):
Screen Shot 2019-07-03 at 5 44 16 PM
Screen Shot 2019-07-03 at 5 44 55 PM

TODO: Change the position of methods bank to the left of the screen with a clickable handle to open and close the methods bank - Will do this in a separate PR next week after Shasha uploads assets for them.

### CODE REVIEW

File: ozaria/site/styles/play/level/tome/spell_palette_entry.sass [hunk 1]
```diff
@@ -0,0 +1,195 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/tome/spell_palette_entry.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/control-bar-view.sass [hunk 2]
```diff
@@ -0,0 +1,181 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/control-bar-view.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/game_dev_track_view.sass [hunk 3]
```diff
@@ -0,0 +1,32 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/game_dev_track_view.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/gold.sass [hunk 4]
```diff
@@ -0,0 +1,51 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/gold.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/hud.sass [hunk 5]
```diff
@@ -0,0 +1,221 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/hud.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/level-dialogue-view.sass [hunk 6]
```diff
@@ -0,0 +1,145 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/level-dialogue-view.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/level/level-playback-view.sass [hunk 7]
```diff
@@ -0,0 +1,163 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/level/level-playback-view.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/play-level-view.sass [hunk 8]
```diff
@@ -0,0 +1,461 @@
+@import "app/styles/mixins"
```
Comment (shubhi1092) [inline ozaria/site/styles/play/play-level-view.sass]: This has been moved from legacy to ozaria folder

File: ozaria/site/styles/play/variables.sass [hunk 9]
```diff
@@ -0,0 +1,3 @@
+$api-bar-width: 279px
```
Comment (shubhi1092) [inline ozaria/site/styles/play/variables.sass]: This has been moved from legacy to ozaria folder, with the only change in the value of $api-bar-width

File: ozaria/site/templates/play/tome/spell_palette_entry_popover.jade [hunk 10]
```diff
@@ -1,116 +1,3 @@
-h4(dir="ltr")
```
Comment (shubhi1092) [inline ozaria/site/templates/play/tome/spell_palette_entry_popover.jade]: This code has been moved below the mixin

File: ozaria/site/templates/play/tome/spell-palette-view.jade [hunk 11]
```diff
@@ -1,8 +1,11 @@
 mixin header(label, name)
-  a.section-header.btn.btn-small.btn-illustrated.btn-warning(role="button", id="#palette-header-" + _.string.slugify(label), data-panel="#palette-tab-" + _.string.slugify(label))
+  div.section-header.text-capitalize(role="button", id="#palette-header-" + _.string.slugify(label), data-panel="#palette-tab-" + _.string.slugify(label))
     span= name
 div.container
   div.left.nano
+    div.header
+      span Command Bank
```
Comment (codeluggage) [inline ozaria/site/templates/play/tome/spell-palette-view.jade]: i18n?

File: ozaria/site/templates/play/tome/spell_palette_entry_popover.jade [hunk 12]
```diff
@@ -150,49 +37,88 @@ mixin argumentEntry(arg)
...
-    p.args(dir="auto")
-      strong
-        span(data-i18n="skill_docs.optional_parameters") Optional Parameters
-        span.spr :
+div.popover-header.closeBtn(role="button" dir="ltr")
+  span.prop-name= doc.shortName
+  if doc.translatedShortName
+    span.translated-name (
+      =doc.translatedShortName
+      | )
```
Comment (codeluggage) [inline ozaria/site/templates/play/tome/spell_palette_entry_popover.jade]: Is this to make an intentional space here? I wonder if that is better than the character code I've been using sometimes.

File: ozaria/site/templates/play/tome/spell_palette_entry_popover.jade [hunk 13]
```diff
@@ -150,49 +37,88 @@ mixin argumentEntry(arg)
...
+  div.glyphicon.glyphicon-chevron-left(style="float: right;")
+
+.popover-body
+  .short-description.rtl-allowed.rtl-right-aligned(dir="auto")
+    p(dir="auto")!= marked(doc.shortDescription || 'Still undocumented, sorry.')
```
Comment (codeluggage) [inline ozaria/site/templates/play/tome/spell_palette_entry_popover.jade]: i18n?

File: ozaria/site/templates/play/tome/spell_palette_entry_popover.jade (hunk 12 above)
Comment (shubhi1092) [reply ozaria/site/templates/play/tome/spell_palette_entry_popover.jade]: This is just to add the closing braces after the translatedShortName without a space (for example: https://stackoverflow.com/questions/27247407/how-to-put-span-in-jade)

**Pull request merged**